test_data/network_params.yaml

```diff
@@ -3,6 +3,7 @@ participants:
     cl_type: lighthouse
     count: 2
   - el_type: ethereumrust
+    cl_image: ethereum_rust
```

Comment:
this is wrong, this is the consensus client

Reply:
Fixed, I was debugging an Issue I had